src/soot/jimple/toolkits/callgraph/InvokeCallSite.java

```diff
@@ -0,0 +1,54 @@
+package soot.jimple.toolkits.callgraph;
+
+import soot.Local;
+import soot.SootMethod;
+import soot.jimple.InstanceInvokeExpr;
+import soot.jimple.Stmt;
+import soot.jimple.toolkits.callgraph.ConstantArrayAnalysis.ArrayTypes;
+
+public class InvokeCallSite {
+	public static final int MUST_BE_NULL = 0;
```

Comment:
Fine with me, unless there are any cross-dependencies that require them to be kept in sync. If they are distinct and just happen to be similar, that's ok.